main/src/ca/uwaterloo/flix/language/phase/jvm/JvmOps.scala

```diff
@@ -258,7 +257,10 @@ object JvmOps {
     * find      =>  m_find
     * length    =>  m_length
     */
-  def getDefMethodNameInNamespaceClass(sym: Symbol.DefnSym): String = "m_" + mangle(sym.name)
+  def getDefMethodNameInNamespaceClass(defn: ReducedAst.Def): String = {
+    if (defn.ann.isExport) defn.sym.name
```

Comment:
Why is it not necessary to mangle the exported name? 

I assume because of restriction on it, but if so we should document it here.

Reply:
The name is checked in safety, so mangling is not required - I can add a comment